{
	"schema_version": "1.6.0",
	"id": "MAL-2024-6077",
	"modified": "2024-10-24T01:01:59Z",
	"published": "2024-06-25T13:43:07Z",
	"aliases": [
		"SNYK-PYTHON-SYSCRYPTPACKAGEV1-5932158"
	],
	"summary": "Malicious code in syscryptpackagev1 (PyPI)",
	"details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "syscryptpackagev1",
				"purl": "pkg:pypi/syscryptpackagev1"
			},
			"versions": [
				"1.1.0",
				"1.0.0"
			],
			"database_specific": {
				"source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/syscryptpackagev1/MAL-2024-6077.json"
			}
		}
	],
	"references": [
		{
			"type": "ARTICLE",
			"url": "https://medium.com/checkmarx-security/the-evolutionary-tale-of-a-persistent-python-threat-d19196d4ed15"
		},
		{
			"type": "ADVISORY",
			"url": "https://security.snyk.io/vuln/SNYK-PYTHON-SYSCRYPTPACKAGEV1-5932158"
		}
	],
	"credits": [
		{
			"name": "ReversingLabs",
			"contact": [
				"https://www.reversinglabs.com"
			],
			"type": "FINDER"
		}
	],
	"database_specific": {
		"malicious-packages-origins": [
			{
				"id": "RLMA-2024-04880",
				"import_time": "2024-06-28T02:50:53.407663465Z",
				"modified_time": "2024-06-25T13:43:07Z",
				"sha256": "ce78f5644e24c509888e54866355fbd414e0c03715b3cbdef8deff550dc0d5e4",
				"source": "reversing-labs",
				"versions": [
					"1.1.0"
				]
			},
			{
				"id": "RLUA-2024-09348",
				"import_time": "2024-10-24T01:00:15.923123618Z",
				"modified_time": "2024-10-16T14:52:12Z",
				"sha256": "df43e4bab14a01ae787121f228783550aa2d89307600b0b21daf4664ce6e5b52",
				"source": "reversing-labs",
				"versions": [
					"1.0.0"
				]
			}
		]
	}
}
